crates/polars-utils/src/cache.rs

```diff
@@ -7,11 +7,9 @@ use ahash::RandomState;
 use bytemuck::allocation::zeroed_vec;
 use bytemuck::Zeroable;
 
-use crate::aliases::PlHashMap;
-
 pub struct CachedFunc<T, R, F> {
```

Comment:
Can we rename this to `FastCachedFunc`. 

It is important that the key is relatively cheap  to compute.